docs/rst/developer_manual/installation/sources/linux.rst

```diff
@@ -376,10 +329,7 @@ Local installation
 Global installation
 -------------------
 
-To install |ddsrecorder| system-wide instead of locally, remove all the flags that
-appear in the configuration steps of :code:`Fast-CDR`, :code:`Fast-DDS`, :code:`Dev-Utils`, :code:`DDS-Router`, and
-:code:`DDS-Recorder`, and change the first in the configuration step of :code:`foonathan_memory_vendor` to the
-following:
+To install |eddsrecord| system-wide instead of locally, remove all the flags that appear in the configuration steps of :code:`Fast-CDR`, :code:`Fast-DDS`, :code:`Dev-Utils`, :code:`DDS-Pipe`, and :code:`DDS-Recorder`, and change the first in the configuration step of :code:`foonathan_memory_vendor` to the following:
```

Comment:
```suggestion
To install |eddsrecord| system-wide instead of locally, remove all the flags that appear in the configuration steps of :code:`Fast-CDR`, :code:`Fast-DDS`, :code:`Dev-Utils`, :code:`DDS-Pipe`, and :code:`DDS-Record`, and change the first in the configuration step of :code:`foonathan_memory_vendor` to the following:
```